views/layouts/default.hbs

```diff
@@ -1,7 +1,10 @@
 <!DOCTYPE html>
-<html lang="{{ locale }}">
+{{!-- MH: not sure locale is actually loading in properly - this is not data removal specific and should be fixed for prod monitor if broken --}}
+{{!-- MH TODO: verify this locale is actually coming through --}}
+<html lang="{{ getLocale }}">
```

Comment:
This should be updated and working now in `main`.

Reply:
👍🏽